component/admin/libraries/jevbuttons.php

```diff
@@ -13,8 +13,9 @@
 // Check to ensure this file is within the rest of the framework
 defined('JPATH_BASE') or die();
 jimport('joomla.html.toolbar.button');
```

Comment:
The auto-loader was taking care of this when I checked, but I've just pushed a change to include it in addition to, to be safer than sorry.